### ISSUE TITLE
Open up our CORS settings to allow JS usage of the Normandy API

### ISSUE THREAD

**Issue by mythmon** [mozilla/normandy#1369]
When we added CORS support to Normandy, [I argued that we should have a whitelist of domains](https://github.com/mozilla/normandy/pull/1202#issuecomment-370585368), instead of allowing any origin. When that happened, I had more in mind things like Delivery Console, and not dashboards, such as those built by S&I.

Hamilton pinged me today, trying to get "around CORS" when working with Normandy. This is a silly situation. We aren't using the same-origin policy as a security mechanism *in this case*, because all GET requests are idempotent, and we don't have sensitive information in our responses. These are the two major attacks the same-origin policy defends against. We also have CSP, which prevents other times of cross-origin attacks.

In short, we should set the allowed origins for CORS to `*`.
